crates/infra/cli/src/commands/check/mod.rs

```diff
@@ -48,6 +51,10 @@ fn check_cargo() -> Result<()> {
     CargoWorkspace::get_command("check")?.run()
 }
 
+fn check_rustdoc() -> Result<()> {
+    CargoWorkspace::get_command("doc")?.run()
```

Comment:
I suggest also appending `--no-deps --document-private-items --examples --bins`. I think we already have one warning from `--bins`.

Reply:
Sure, changed in 8aa5435c48838c345ea6d697fef100e4c0a425f3.